source/instructors/authoring/guides/markdown_content.rst

```diff
@@ -30,6 +30,16 @@ Writing content
 ***************
 Write content in the main content area.
 
+You can also set hotkeys for a range of commonly used editing functions. 
+
+- Bold selected text
+- Enumerate selected text
+- Order selected text
+- Make selected text as a code 
```

Comment:
Make selected text as a code -> Make selected text a code block

Reply:
Did wonder but just took from what was given in the issue :)